Snippets/CloudServicesHost/CloudServicesHost_7/HostingAndConfiguration.cs

```diff
@@ -61,3 +62,18 @@ protected void Application_Start()
 }
 
 #endregion
+
+namespace AsAHost
+{
+    #region AsAHost
+
+    public class EndpointConfig : IConfigureThisHost
```

Comment:
Is `EndpointConfig` a good name? We've got `IConfigureThisEndpoint` and class implementing it is often called `EndpointConfig`. Would `EndpointHostConfit` be more clear?